### DISCUSSION TITLE
申请一下退款，已购买专业版的邮箱是：953735548@qq.com

### DISCUSSION THREAD

**Discussion by 8coooo** [getlantern/lantern-forum-cn#258] [Q&A]
还请帮忙退一下款，谢谢！

**Comment by MaxMKMax** [comment]
@8coooo 您好, 请问您可以向我们的支付团队发送电子邮件吗？您可以提供您的收据. 因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。

电子邮件：support@getlantern.org

**Reply by MaxMKMax** [reply]
<img width="110" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/8c3ac95d-08da-4103-9718-d425e3daabfb">

**Reply by 8coooo** [reply]
回复邮件了

**Comment by 8coooo** [comment]
------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
Date:&nbsp;Tue, Mar 19, 2024 03:30 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [getlantern/lantern-forum-cn] ***@***.*** (Discussion #258)





 
@8coooo 好的. 我帮您安排. 请问您可以向我提供您的收据吗？

**Reply by MaxMKMax** [reply]
@8coooo 嗯嗯. 好的. 如果有什么问题还没搞定, 您再来找我. 我来帮您!

**Reply by 8coooo** [reply]
![Uploading 81240e3e55a9190a42649be8855ef03.jpg…]()

**Reply by 8coooo** [reply]
![81240e3e55a9190a42649be8855ef03](https://github.com/getlantern/lantern-forum-cn/assets/37218653/becfd512-a011-489c-9d48-1539cdf702c4)

**Reply by 8coooo** [reply]
您好，我把付款截图发到这里了，还请帮忙处理一下退款，我一周以前就给支付团队发过邮件了， 没有得到任何回复。

**Reply by MaxMKMax** [reply]
@8coooo 好的明白了. 因为我不是客服部门的所以无法帮您检查情况. 我今晚帮您对接负责人. 请放心.
对了, 请您也提供我您的注册邮箱! 谢谢！

**Comment by 8coooo** [comment]
***@***.***




------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
Date:&nbsp;Mon, Mar 25, 2024 09:41 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [getlantern/lantern-forum-cn] ***@***.*** (Discussion #258)





 
@8coooo 好的明白了. 因为我不是客服部门的所以无法帮您检查情况. 我今晚帮您对接负责人. 请放心.
 对了, 请您也提供我您的注册邮箱! 谢谢！

**Comment by MaxMKMax** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/ce7a3ec2-9731-4e2c-95d3-48990b79e99b)

抱歉. 我好像看不到您的邮箱. 被屏蔽了. 请您这样写邮箱给我. 空格写. 
s u p p ort@ get lanter n .org

**Reply by MaxMKMax** [reply]
@8coooo 您的邮箱是这个吗?
953735548@qq.com

**Comment by 8coooo** [comment]
***@***.***




------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
Date:&nbsp;Mon, Apr 1, 2024 04:48 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [getlantern/lantern-forum-cn] ***@***.*** (Discussion #258)





 
@8coooo 您的邮箱是这个吗?
 ***@***.***

**Comment by 8coooo** [comment]
9 5 3 7 3 5 5 4 8 @ q q .com




------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
Date:&nbsp;Fri, Apr 12, 2024 07:30 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [getlantern/lantern-forum-cn] ***@***.*** (Discussion #258)





 
@8coooo 抱歉. 我好像看不到您的邮箱. 被屏蔽了. 请您这样写邮箱给我. 空格写.
 s u p p ort@ get lanter n .org

**Comment by MaxMKMax** [comment]
@8coooo 您好. 已经帮您提交问题上去了. 谢谢您支持蓝灯. 希望以后还能再看到您! 欢迎您随时回来用蓝灯!
您决定退款钱要不再试试看我们的最新版本. 如果没问题我们希望您继续成为我们尊贵的用户. 谢谢您!

https://github.com/getlantern/lantern
